data-jdbc/src/test/groovy/io/micronaut/data/jdbc/postgres/PostgresColumnTransformerSpec.groovy

```diff
@@ -88,7 +88,7 @@ class Rider {
 
     String name
 
-    @ColumnTransformer(read = "to_char(@.finish_time, 'PTHH24HMIM')")
+    @ColumnTransformer(read = "to_char(finish_time, 'PTHH24HMIM')")
```

Comment:
Is it needed?

Reply:
In prev PR I thought `@.` is needed for `@ColumnTransformer` but it works without it so just reverting change here.

Reply:
ok